curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which property changes the background color in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+`bg-color`
 
 ---
 
-Placeholder distractor 2
+`color-bg`
 
 ---
 
-Placeholder distractor 3
+`fill-color`
 
 #### --answer--
 
-Placeholder answer
+`background-color`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which CSS property controls the width of an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+`size`
 
 ---
 
-Placeholder distractor 2
+`dimension`
 
 ---
 
-Placeholder distractor 3
+`area`
 
 #### --answer--
 
-Placeholder answer
+`width`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which tool can help inspect element styles in a web browser?
 
 #### --distractors--
 
-Placeholder distractor 1
+Adobe Photoshop
 
 ---
 
-Placeholder distractor 2
+Microsoft Word
 
 ---
 
-Placeholder distractor 3
+Notepad
 
 #### --answer--
 
-Placeholder answer
+Chrome DevTools
```

Comment:
I'm only allowing this because you did not say mention Chrome specifically in the question.